assembl/static2/js/app/components/common/ChatFrame.jsx

```diff
@@ -2,12 +2,7 @@ import React from 'react';
 import { Translate } from 'react-redux-i18n';
 import { OverlayTrigger, Tooltip } from 'react-bootstrap';
 import { connect } from 'react-redux';
-
-const varPath = (root, path) => {
-  return path.split('.').reduce((value, nextKey) => {
-    return value && value[nextKey];
-  }, root);
-};
+import lodashGet from 'lodash/get';
```

Comment:
Just `get` would do 😀

Reply:
we already have a `get` function used everywhere, I think this would have caused confusion at some point